### ISSUE TITLE
xmms-ctl marked as "newer than repo" but they are the same

### ISSUE THREAD

**Issue by nihilismus** [sbopkg/sbopkg#28]
```
root# sbopkg -c
Listing installed SBo repository for Slackware 14.2 packages and flagging
potential updates...

xmms-ctl:
  INSTALLED PACKAGE IS NEWER THAN REPO
  Installed version:  xmms-ctl-4158c718bf68cd8653a0aee93afe65202e483ef9-x86_64-1_SBo
  Repo version:       xmms-ctl-4158c718bf68cd8653a0aee93afe65202e483ef9-x86_64-1_SBo

Potential update list complete.
```

**Comment by willysr** [comment]
Hi

it seems this issue is the same as https://github.com/sbopkg/sbopkg/issues/24

**Comment by willysr** [comment]
This should be fixed by now after SBo's latest update.

**Comment by willysr** [comment]
This should be completely fixed in c24c2dd5d6917c05815f1b3f70f552424408bbb5
